src/components/Templates/Pages/CommonPage/CommonPage.component.tsx

```diff
@@ -0,0 +1,40 @@
+import Box from '@mui/material/Box';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { categories } from '../../../../config/config';
+import { IProduct } from '../../../../types/models/Product';
+import { Loading } from '../../../Molucules/Loading';
+import { ProductCardContainer } from '../../ProductCardConatiner';
+import { SectionLayout } from '../../SectionLayout';
+
+const CommonPage = () => {
+  const { cat } = useParams();
+  const category = categories.find((category) => category.link === cat);
+
+  const [products, setProducts] = useState<IProduct[]>([]);
+  useEffect(() => {
+    axios
+      .get(`https://fakestoreapi.com/products/category/${category?.title}`)
+      .then((res) => {
+        setProducts(res.data);
+      })
+      .catch((err) => {
+        console.log('error', err);
+      });
+  }, []);
+
+  return (
+    <Box width={1} maxWidth={1100} mx='auto'>
```

Comment:
@siroshan 

- see if you can extract this `Box` component to a seperate `PageLayout` that can be used in `HomePage` as well—they are similar, so you should be able to wrap it in a Layout so that we can update them all at once when needed!

- also let's rename this to `CategoryPage` which is much clear!